tests/test_wallet.py

```diff
@@ -136,3 +137,13 @@ def test_fungible_token_transfer_planetmint():
     assert len(message["outputs"]) == 2
     assert message["outputs"][0]["amount"] == "30"
     assert message["outputs"][1]["amount"] == "70"
+
+
+def test_get_address():
+    wallet = SoftwareWallet()
```

Comment:
Can we init `SoftwareWallet()` with a fixed private portion, so that we know `liquid_address` and `planemint_address` beforehand? So that we can assert on the value and not only the length?

Reply:
This is done by setting the seed on the API.

Reply:
wanna do that or leave the test case on `len()`?

Reply:
we leave it at that for now